invokeai/frontend/web/src/features/nodes/components/ui/ReloadSchemaButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { useAppDispatch } from 'app/store/storeHooks';
+import IAIIconButton from 'common/components/IAIIconButton';
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { BiRefresh } from 'react-icons/bi';
+import { receivedOpenAPISchema } from 'services/api/thunks/schema';
+
+export default function ReloadSchemaButton() {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+
+  const handleReloadSchema = useCallback(() => {
+    dispatch(receivedOpenAPISchema());
+  }, [dispatch]);
+
+  return (
+    <IAIIconButton
+      icon={<BiRefresh />}
```

Comment:
Can we use the corresponding FontAwesome icon here? Trying to use FontAwesome everywhere so we have both consistency in style and in sizing - different icon packs have their own unique size, padding and style, that makes using different packs look wonky.